docker-vms/freeswitch/update_config_from_env.sh

```diff
@@ -0,0 +1,13 @@
+set -e
+
+# 01_sip_dialplan.xml
+FILE="dialplan/public/01_sip_dialplan.xml"
+sed -i 's/{{number}}/'"$SIP_NUMBER"'/' "$FILE"
```

Comment:
We probably want `s/.../.../g` so that if there are multiple copies of `{{foo}}` on a line they all get replaced.